MPF.Core/Modules/XboxBackupCreator/Parameters.cs

```diff
@@ -0,0 +1,684 @@
+            switch (this.Type)
+            {
+                case MediaType.DVD:
+                    if (!File.Exists($"{basePath}_logs.zip") || !preCheck)
+                    {
+                        string baseDir = Path.GetDirectoryName(basePath) + Path.DirectorySeparatorChar;
+                        string? logPath = GetLogName(baseDir);
+                        if (string.IsNullOrEmpty(logPath))
+                            missingFiles.Add($"{baseDir}Log.txt");
+                        if (!File.Exists($"{baseDir}DMI.bin"))
+                            missingFiles.Add($"{baseDir}DMI.bin");
+                        if (!File.Exists($"{baseDir}PFI.bin"))
+                            missingFiles.Add($"{baseDir}PFI.bin");
+                        if (!File.Exists($"{baseDir}SS.bin"))
+                            missingFiles.Add($"{baseDir}SS.bin");
+
+                        // Not required from XBC
+                        //if (!File.Exists($"{basePath}.dvd"))
+                        //    missingFiles.Add($"{basePath}.dvd");
+                    }
+
+                    break;
+
+                default:
+                    missingFiles.Add("Media and system combination not supported for XboxBackupCreator");
+                    break;
+            }
+
+            return (!missingFiles.Any(), missingFiles);
+        }
+
+        /// <inheritdoc/>
+        public override void GenerateSubmissionInfo(SubmissionInfo info, Options options, string basePath, Drive? drive, bool includeArtifacts)
+        {
+            // Ensure that required sections exist
+            info = Builder.EnsureAllSections(info);
+
+            // Get base directory
+            string baseDir = Path.GetDirectoryName(basePath) + Path.DirectorySeparatorChar;
+
+            // Get log filename
+            string? logPath = GetLogName(baseDir);
+            if (string.IsNullOrEmpty(logPath))
+                return;
+
+            // XBC dump info
+            info.DumpingInfo!.DumpingProgram = $"{EnumConverter.LongName(this.InternalProgram)} {GetVersion(logPath) ?? "Unknown Version"}";
+            info.DumpingInfo.DumpingDate = InfoTool.GetFileModifiedDate(logPath)?.ToString("yyyy-MM-dd HH:mm:ss");
+            info.DumpingInfo.Model = GetDrive(logPath) ?? "Unknown Drive";
+
+            // Look for read errors
+            if (GetReadErrors(logPath, out long readErrors))
+                info.CommonDiscInfo!.ErrorsCount = readErrors == -1 ? "Error retrieving error count" : readErrors.ToString();
+
+            // Extract info based generically on MediaType
+            switch (this.Type)
+            {
+                case MediaType.DVD:
+
+                    // Get PVD from ISO (Currently only for Xbox360)
+                    if (this.System != RedumpSystem.MicrosoftXbox360)
+                    {
+                        if (GetPVD(basePath + ".iso", out string? pvd))
+                            info.Extras!.PVD = pvd;
+                    }
```

Comment:
If this is only for X360, then you should have a separate switch block for the system, similar to other modules